runtime/kusama/src/lib.rs

```diff
@@ -681,6 +681,9 @@ parameter_types! {
 	pub const BountyValueMinimum: Balance = 200 * CENTS;
 	pub const MaxApprovals: u32 = 100;
 	pub const MaxAuthorities: u32 = 100_000;
+	pub const MaxKeys: u32 = 10_000;
+	pub const MaxPeerInHeartbeats: u32 = 10_000;
+	pub const MaxPeerIdEncodingSize: u32 = 1_000;
```

Comment:
those numbers are probably way bigger than the real limit. But we can make them more precise later.

Reply:
Thank you. Do you mind adding the labels? I don’t have write access to do it myself unfortunately. Thanks!